recipe/meta.yaml

```diff
@@ -180,7 +181,8 @@ outputs:
         - curl -O https://raw.githubusercontent.com/inducer/pyopencl/v2021.2.13/examples/demo.py
         - export PYOPENCL_CTX=intel  # [unix]
         - set PYOPENCL_CTX=intel     # [win]
-        - python demo.py
+        # TODO: enable this test for win
+        - python demo.py             # [not win]
```

Comment:
This fails with OUT_OF_HOST_MEMORY.

cc @napetrov

Reply:
There is workaround that can be implemented with setting TBB_DLL_PATH

Example on workaround
https://github.com/IntelPython/dpctl/blob/master/.github/workflows/conda-package.yml#L221-L250

Will be fixing in package activation script